koku/masu/database/report_manifest_db_accessor.py

```diff
@@ -65,9 +65,8 @@ def mark_manifests_as_completed(self, manifest_list):
         if manifest_list:
             bulk_manifest_query = self._get_db_obj_query().filter(id__in=manifest_list)
             for manifest in bulk_manifest_query:
-                if not manifest.manifest_completed_datetime:
-                    manifest.manifest_completed_datetime = completed_datetime
-                    manifest.save()
+                manifest.manifest_completed_datetime = completed_datetime
```

Comment:
Why remove the condition here?

Reply:
I had added that if statement in this [previous pr](https://github.com/project-koku/koku/pull/4044)
But with it, the `manifest_completed_datetime` is not updated when there’s still more processing done for the same manifest id. I noticed this on manifests with (start, end) date pairs with ranges wider than the `TRINO_DATE_STEP` set in the env. 

I also now think it would prevent update on a re-summary.

Reply:
I had the same question as Andrew but I think this makes sense.